source/frontend/src/components/common/form/FastDatePicker.tsx

```diff
@@ -130,6 +133,7 @@ const FormikDatePicker: FunctionComponent<React.PropsWithChildren<FastDatePicker
         maxDate={maxDate}
         {...rest}
         popperModifiers={popperModifiers}
+        popperPlacement={popperPlacement !== undefined ? popperPlacement : 'top-end'}
```

Comment:
nit: we use the "exists" function to check for null or undefined. Please update this like so:

```tsx
popperPlacement={exists(popperPlacement) ? popperPlacement : 'top-end'}
```